packages/react-components/src/templates/SharedResearchOutputGrantDocument.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ComponentProps } from 'react';
+import { css } from '@emotion/react';
+import { ResearchOutputResponse } from '@asap-hub/model';
+
+import { Card, Headline2 } from '../atoms';
+import { perRem } from '../pixels';
+import { contentSidePaddingWithNavigation } from '../layout';
+import { BackLink, CtaCard, TagList } from '../molecules';
+import {
+  RichText,
+  SharedResearchAdditionalInformationCard,
+  SharedResearchOutputHeaderCard,
+} from '../organisms';
+import { createMailTo } from '../mail';
+
+const containerStyles = css({
+  padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(8)}`,
+});
+
+const cardsStyles = css({
+  display: 'grid',
+  rowGap: `${36 / perRem}em`,
+});
+
+type SharedResearchOutputGrantDocumentProps = Pick<
+  ResearchOutputResponse,
+  'description' | 'tags' | 'contactEmails'
+> &
+  ComponentProps<typeof SharedResearchOutputHeaderCard> & {
+    backHref: string;
+  } & ComponentProps<typeof SharedResearchAdditionalInformationCard>;
+
+const SharedResearchOutputGrantDocument: React.FC<SharedResearchOutputGrantDocumentProps> =
```

Comment:
I wonder if this name should reflect the difference between the templates which is an expanded description SharedResearchOutputRTFDescription